packages/core/clients/fallbackKernelAccountClient.ts

```diff
@@ -11,50 +11,79 @@ export const createFallbackKernelAccountClient = <
 >(
     clients: Array<
         KernelAccountClient<TEntryPoint, TTransport, TChain, TSmartAccount>
-    >
+    >,
+    onError?: (error: Error, clientUrl: string) => Promise<void>
 ): KernelAccountClient<TEntryPoint, TTransport, TChain, TSmartAccount> => {
+    // Map to store methods extended at runtime.
+    // biome-ignore lint/suspicious/noExplicitAny: <explanation>
+    const extendedMethods = new Map<PropertyKey, any>()
```

Comment:
@adnpark just curious, is the extendMethods object needed?  I don't see us ever setting any values on it

Reply:
yeah it makes sense. we don't need it.